packages/data-provider/src/utils/formatUtils.ts

```diff
@@ -115,6 +115,10 @@ export const deepCompare = (obj1: unknown, obj2: unknown): boolean => {
 	return true;
 };
 
+export const isNotNull = <T>(value: T): value is Exclude<T, null> => {
+	return value !== null;
+};
+
```

Comment:
This function hardly seems necessary... but we can add it since a lot of the methods here are also not needed but seem convenient.

In the newer versions of TS you don't need to declare `value is ...` in the type signature if the function returns a boolean after narrowing the type. The TS compiler will infer the type narrowing correctly. This is the preferred way to write these function signatures since developers can write the type narrowing wrong, or the function can be modified later causing the type narrowing to be invalid. Best to let the compiler infer it. Generally, consider `value is X` similar to asserting a type `as X`.